ui/app/components/ui/popover/popover.header.component.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+const PopOverheader = ({ title, onClose }) => (
+  <div className="popover-header">
+    <h2 className="popover-header__heading">{title}</h2>
+    <a href="#" className="popover-header__close" onClick={onClose}>
```

Comment:
Could this be made a button instead of an anchor? It seems like more of a button than an anchor to me.